docs/content/en/docs-dev/contribution-guidelines/development.md

```diff
@@ -11,6 +11,7 @@ description: >
 - [Go 1.19](https://go.dev/)
 - [Docker Decktop](https://www.docker.com/products/docker-desktop/)
```

Comment:
not related to this PR but could you change this required tool to docker instead of docker desktop